themes/color-black.yaml

```diff
@@ -4,19 +4,76 @@ global:
     dark_color:                 [0.250,0.250,0.250]
     mid_color:                  [0.500,0.500,0.500]
     midlight_color:             [0.625,0.625,0.625]
+    light:                      [0.690,0.690,0.690]
```

Comment:
this needs a better name, with a `_color` postfix. `extra_light_color`? And changed in all the files that  use the global, please.